release-notes/v0.9.0.md

```diff
@@ -0,0 +1,76 @@
+
+Here follows a summary of notable changes, and the non backward compatible changes.
+The full change log is available [here](https://github.com/cucumber/godog/blob/master/CHANGELOG.md).
+
+
+Notable Changes
+----------------
+
+Most importantly, note that the gherkin core is changed to [gherkin-go](https://github.com/cucumber/gherkin-go/releases/tag/v9.2.0).
+
+
+
+Non backward compatible changes
+-------------------------------
+
+### Install godog
+With the change of dependencies for godog, which relies on `go modules`, installing godog now requires go modules to be active.
+
+If you are running `within the $GOPATH`, you would need to install godog like this:
+`GO111MODULE=on go get github.com/cucumber/godog/cmd/godog@v0.9.0`
+- Adding `GO111MODULE=on` will allow go get and go mod to work together as intended.
+- Adding `@v0.9.0` will install v0.9.0 specifically instead of master.
+
+If you are running `outside of the $GOPATH`, you should still specify a version.
+
+I you encounter this error, please try adding `GO111MODULE=on`:
+```
+/go# go get github.com/cucumber/godog/cmd/godog
+package github.com/cucumber/gherkin-go/v9: cannot find package "github.com/cucumber/gherkin-go/v9" in any of:
+	/usr/local/go/src/github.com/cucumber/gherkin-go/v9 (from $GOROOT)
+	/go/src/github.com/cucumber/gherkin-go/v9 (from $GOPATH)
+package github.com/cucumber/messages-go/v9: cannot find package "github.com/cucumber/messages-go/v9" in any of:
+	/usr/local/go/src/github.com/cucumber/messages-go/v9 (from $GOROOT)
+	/go/src/github.com/cucumber/messages-go/v9 (from $GOPATH)
+```
+
+
+### Gherkin Core
+The following types have been switched for [messages-go](https://github.com/cucumber/messages-go).
+
+|           old           |                     new                     |
+| ----------------------- | ------------------------------------------- |
+| gherkin.Feature         | messages.GherkinDocument                    |
+| gherkin.Scenario        | messages.Pickle                             |
+| gherkin.ScenarioOutline | messages.Pickle                             |
+| gherkin.Step            | messages.Pickle_PickleStep                  |
+| gherkin.DocString       | messages.PickleStepArgument_PickleDocString |
+| gherkin.DataTable       | messages.PickleStepArgument_PickleTable     |
+
+
+### Step Defintions
+- `StepDef` has been renamed to `StepDefinition`
+- Steps that earlier accepted `*gherkin.DocString` or `*gherkin.DataTable` needs to be updated to use `*messages.PickleStepArgument_PickleDocString` resp. `*messages.PickleStepArgument_PickleTable`.
+[Example](https://github.com/cucumber/godog/pull/240/files#diff-a5f59d298843b731ff8d2f9c670303ff).
+
+
+### Hooks
+The updated hooks can be found [here](https://github.com/cucumber/godog/blob/b62eb13ee70c9f0f732b694b39bde9670051bac7/suite.go#L251).
+- `BeforeFeature` and `AfterFeature` hooks now accept `*messages.GherkinDocument` instead of `*gherkin.Feature`
```

Comment:
Can you add a warning about these being deprecated in the future?

Reply:
The Feature Hooks?

Reply:
Added deprecation notice, check if you think it's good enough :)